Applications/Spire/Source/Ui/Button.cpp

```diff
@@ -113,7 +121,7 @@ Button* Spire::make_label_button(const QString& label, QWidget* parent) {
     set(TextColor(QColor::fromRgb(0, 0, 0))).
```

Comment:
This text color is not in the specs, remove it from this.

Reply:
I can remove it for now but the proper text color will depend on the inclusion of the new Label component.